lib/Twig/Test/IntegrationTestCase.php

```diff
@@ -176,7 +178,14 @@ protected function doIntegrationTest($file, $message, $condition, $templates, $e
             $p->setAccessible(true);
             $p->setValue($twig, '__TwigTemplate_'.hash('sha256', uniqid(mt_rand(), true), false).'_');
 
+            $deprecations = array();
             try {
+                set_error_handler(function ($type, $msg) use (&$deprecations) {
+                    if (E_USER_DEPRECATED === $type) {
+                        $deprecations[] = $msg;
+                    }
```

Comment:
the original error handler should be called for any other error, to make the rendering fail. Otherwise, this is equivalent to silencing all non-deprecation errors in tests (defeating the goal of the test)

Reply:
fixed